prediction_market_agent/agents/think_thoroughly_agent/prompts.py

```diff
@@ -83,21 +83,20 @@
 The report must not be longer than 1000 words.
 """
 
-FINAL_DECISION_PROMPT = """
-Your task is to determine the probability of a SCENARIO being answered 'Yes' or 'No'.
-SCENARIOS_WITH_PROBABILITIES is provided, which contains a set of {number_of_scenarios} SCENARIOs.
-The object SCENARIOS_WITH_PROBABILITIES contains a list of tuple, where each tuple contains, as its
-first element, an scenario, and as its second entry, a probability dictionary, having properties such
-as p_yes (probability that the scenario is true), p_no (probability that the scenario is false),
-confidence (confidence level in the values of p_yes and p_no).
-You should determine the probability of the SCENARIO SCENARIO_TO_ASSESS being true, 
-considering the probabilities of the other related SCENARIOs.
+FINAL_DECISION_PROMPT = """Your task is to determine the probability of a SCENARIO being answered 'Yes' or 'No'. SCENARIOS_WITH_PROBABILITIES is provided, which contains a set of {number_of_scenarios} SCENARIOs. The object SCENARIOS_WITH_PROBABILITIES contains a list of tuple, where each tuple contains, as its first element, an scenario, and as its second entry, a probability dictionary, having properties such as p_yes (probability that the scenario is true), p_no (probability that the scenario is false), confidence (confidence level in the values of p_yes and p_no). 
 
 SCENARIOS_WITH_PROBABILITIES: 
 {scenarios_with_probabilities}
 
-SCENARIO_TO_ASSESS: {scenario_to_assess}
-"""
+CORRELATED_MARKETS is also provided, which contains a list of scenarios that are semantically similar to SCENARIO_TO_ASSESS. The object RELATED_MARKETS contains a list of dictionaries, where each dictionary has properties such as question_title, as p_yes (probability that the scenario is true) and p_no (probability that the scenario is false).
+
+CORRELATED_MARKETS: 
+{correlated_markets}
```

Comment:
Main part - added correlated_markets to the prompt as reference.

Reply:
Are they necessary correlated? Similar in the question for sure, but they don't have to be correlated. Or are you planning to do some filtering only for corelated ones?